charts/trino/values.yaml

```diff
@@ -247,6 +254,17 @@ worker:
     # failureThreshold: 6
     # successThreshold: 1
 
+  lifecycle: {}
+    # preStop:
+    #   exec:
+    #     command: ["/bin/sh", "-c", "curl -v -X PUT -d '\"SHUTTING_DOWN\"' -H \"Content-type: application/json\" http://localhost:8081/v1/info/state"]
+    # https://trino.io/docs/current/admin/graceful-shutdown.html#admin-graceful-shutdown--page-root
+
+  terminationGracePeriodSeconds: ""
+  # Sleep for shutdown.grace-period, which defaults to 2 minutes.
```

Comment:
What does this comment apply to?

Reply:
I took this comment from [Trino docs](https://trino.io/docs/current/admin/graceful-shutdown.html#shutdown-behavior).

Reply:
But why is it here?

Reply:
I put it there to help people define the value correctly because it should be equal to or greater than the Trino `shutdown.grace-period`, but I can remove it if you prefer.

Reply:
That's ok, but you should phrase it appropriately. Be explicit.

Reply:
I rephrase to be explicit.
What do you think of how it is now?

Reply:
That's better, thanks! For comments without any examples, I see they're usually placed before the property. Also, can you mention in which chart section users can configure the `shutdown.grace-period` property? The distinction between chart and Trino properties is not very clear.

Reply:
Sure, I mention where to configure the `shutdown.grace-period` and allow the operation in the `accessControl`.
 
```git
diff --git a/charts/trino/values.yaml b/charts/trino/values.yaml
index dadd46f..3110313 100644
--- a/charts/trino/values.yaml
+++ b/charts/trino/values.yaml
@@ -258,10 +258,13 @@ worker:
     # preStop:
     #   exec:
     #     command: ["/bin/sh", "-c", "curl -v -X PUT -d '\"SHUTTING_DOWN\"' -H \"Content-type: application/json\" http://localhost:8081/v1/info/state"]
     # https://trino.io/docs/current/admin/graceful-shutdown.html#admin-graceful-shutdown--page-root
+    # access control does not allow graceful shutdowns by default, to configure check the accessControl section
 
   terminationGracePeriodSeconds: "30"
   # The terminationGracePeriodSeconds should be greater or equal to shutdown.grace-period.
+  # To configure shutdown.grace-period, check the lifecycle section.
 
   nodeSelector: {}
```